### PR TITLE
simple project list

### PR DESCRIPTION
**Pull request by xi**
This adds a very simple project list.

Since the project-detail view is already in core, it may be a good idea to move the project-list view to core, too. But I guess we can first see what we have to add to it here and move it later if it still seems like a good idea.

### CODE REVIEW

**Comment by xi** [comment]
I just realized that there already was a project-list as part of the dashboard. We should either merge them or give them more specific names.

File: meinberlin/assets/scss/_layout.scss [hunk 1]
```diff
@@ -36,5 +40,10 @@
             margin-left: 0;
             margin-right: 0;
         }
+
+        .l-tile-4.l-tile-4 {
```
Comment (vellip) [inline meinberlin/assets/scss/_layout.scss]: this appears to be a bit hacky, i think?

Comment (xi) [inline meinberlin/assets/scss/_layout.scss]: See comment above.

**Comment by vellip** [comment]
@xi, @2e2a what do you think of [this](https://github.com/liqd/a4-meinberlin/pull/69/commits/bb122f64b19047e3c4a071b01824cd35e710d9c5) way to get project urls from a4 and our owns together?

**Comment by 2e2a** [comment]
> @xi, @2e2a what do you think of this way to get project urls from a4 and our owns together? At least for now, before we move it into a4 directly.

Makes sense for me.

**Comment by 2e2a** [comment]
But import order needs to be fixed.

File: apps/projects/urls.py [hunk 2]
```diff
@@ -6,3 +7,5 @@
     url(r'^$', views.ProjectListView.as_view(),
         name='project-list'),
 ]
+
+urlpatterns += a4_projects_urls
```
Comment (xi) [inline apps/projects/urls.py]: Definitely nicer than my version!

**Pull request merged**